neptune/new/exceptions.py

```diff
@@ -574,21 +574,26 @@ class NeptuneWrongInitParametersException(NeptuneException):
 
 
 class NeptuneInitParametersCollision(NeptuneWrongInitParametersException):
-    def __init__(self, parameter1, parameter2):
+    def __init__(self, parameter1, parameter2, method_name):
         self.parameter1 = parameter1
         self.parameter2 = parameter2
+        self.method_name = method_name
         message = """
 {h1}
-----NeptuneRunResumeAndCustomIdCollision-----------------------------------------
+----NeptuneInitParametersCollision-----------------------------------------
 {end}
-The {python}{parameter1}{end} and {python}{parameter2}{end} parameters of the {python}init(){end} method are mutually exclusive.
+The {python}{parameter1}{end} and {python}{parameter2}{end} parameters of the {python}{method_name}(){end} method are mutually exclusive.
```

Comment:
Exception still has `Init` in its name. Maybe we should have `NeptuneInitParametersCollision` inheritating from `NeptuneParametersCollision` to not break backward compatibility and still have explanatory name.

Reply:
I just wanted to have more generic method name, but only for `init_run`, `init_model` and so on.
I don't think that `Init` in exception name could be a problem.
I don't even expect other places where we could have problems with such backward compatybility.